esm-samples/jsapi-angular-cli/README.md

```diff
@@ -5,73 +5,45 @@ This repo demonstrates how to use [@arcgis/core](https://www.npmjs.com/package/@
 ---
 **Known issues**
 
-* It's currently recommended to set `buildOptimizer: false` in `angular.json`. This prevents WebGL errors such as `WebGL: INVALID_OPERATION` and `WebGL: INVALID_ENUM`. It may increase your on-disk and loaded application size.
+* We are aware of a general issue with `Unhandled Promise Rejection` errors. They aren't specific to Angular and shouldn't affect ArcGIS JS API functionality. Upgrade to `11.2.5`+ and switch the `tsconfig.target` to es2017.
 
-* We are aware of a general issue with `Unhandled Promise Rejection` errors. They aren't specific to Angular and shouldn't affect ArcGIS JS API functionality. 
+* if you are seeing CommonJS or AMD dependency warnings you can supress them in your build output thought a property setting in `angular.json`. Suppressing won't affect functionality. Also, consider upgrading by running `npm i @arcgis/core@latest`.
 
----
-
-## Get Started
-
-**Step 1** - Run `npm install`. 
-
-**Step 2** Copy assets - There are several small changes to this repo that enable the JS API to work. In `angular.json` copy the JS API assets folder to `/assets`. And, under build > options > allowedCommonJsDependencies you can add `moment` to the array to suppress `CommonJS or AMD dependencies` warnings in dev builds.
+*angular.json*
 
 ```json
   "allowedCommonJsDependencies": [
     "moment"
   ],
-  "assets": [
-    {
-      "glob": "**/*",
-      "input": "node_modules/@arcgis/core/assets",
-      "output": "/assets/"
-    },
-    "src/favicon.ico",
-    "src/assets"
-  ],
```

Comment:
Will this be available elsewhere for those that do wish to use local assets?

Reply:
That's just a default angular assets setting, it's not related to JS API.

Reply:
Yes, but it was hard to find 😉